### ISSUE TITLE
Staging Review finding: Success alert doesn't appear until after focus moves (likely bug)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82519]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** List and Loop
**Feature:** Reusable multi-page

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When editing a list-and-loop response, when returning to "Review your employers" no success alert is displayed. However, when keyboard focus moves to another element on the screen (eg. tabbing to Back), the success alert appears.

I'm guessing this is a bug that snuck in.

Note also that the X button to close the alert does not appear to be working. Also looks like a bug, not sure if related since both touch focus management.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Add an employer.
2. On "Review your employers," click Edit.
3. Make an edit to the employer, and click Continue to get back to the review page.
4. Note that no success alert is visible.
5. On "Review your employers," press [tab] until focus moves to Back.
6. Note that the success alert appears on screen.

</details>

## Recommended action

My ideal behavior would be for the success alert to be on screen when landing on the page, and for focus to go directly on the success alert to ensure it's the first thing announced to screen reader users.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by rhasselle-oddball** [comment]
can be tested at https://staging.va.gov/mock-form-patterns/introduction

**Comment by jeana-adhoc** [comment]
This was addressed in this ticket: https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1344
